examples/mlserver/client.py

```diff
@@ -0,0 +1,27 @@
+import requests, threading
+
+NUM_THREADS = 2
+URL = "http://localhost:8080/v2/models/text-classification-model/infer"
+sentences = ["I hate using GPUs for inference", "I love using DeepSparse on CPUs"] * 100
+
+def tfunc(text):
+    inference_request = {
+        "inputs": [
+            {
+                "name": "sequences",
+                "shape": [1],
+                "datatype": "BYTES",
+                "data": [text],
+            },
+        ]
+    }   
+    resp = requests.post(URL, json=inference_request).json()
+    for output in resp["outputs"]:
+        print(output["data"])
```

Comment:
executing a list printout while multithreaded may cause a race condition, any reason to not return the value and print in sequence at the end? (ie consider thread 1 and thread 2 happen to execute *exactly* at the same time, they will print their lines at the same time and might not tell which is which)